Samples/office-add-in-save-custom-settings/README.md

```diff
@@ -0,0 +1,120 @@
+---
+page_type: sample
+urlFragment: office-add-in-save-custom-settings
+products:
+- office-excel
+- office-word
+- office-powerpoint
+- office
+languages:
+- javascript
+extensions:
+  contentType: samples
+  technologies:
+  - Add-ins
+  createdDate: "08/26/2022 10:00:00 AM"
+description: "This sample shows how to save custom settings in Office Add-in."
+---
+
+# Save custom settings in your Office Add-in
+
+This sample shows how to save custom settings inside an Office Add-in. The add-in stores data as key/value pairs, using the JavaScript API for Office property bag, browser cookies, web storage (**localStorage** and **sessionStorage**), or by storing the data in a hidden div in the document.
+
+## Applies to
+
+- Excel, Word, and PowerPoint on Windows, Mac, and in a browser.
+
+## Prerequisites
+
+- Microsoft 365 - You can get a [free developer sandbox](https://developer.microsoft.com/microsoft-365/dev-program#Subscription) that provides a renewable 90-day Microsoft 365 E5 developer subscription.
+
+## Run the sample
```

Comment:
I followed the steps, but the add-in icon renders as a broken image and clicking the add-in gives me a 404 (File not found) error. Is GitHub Pages enabled on your branch by any chance?

Reply:
The task pane loads correctly when the sample is run locally on a Windows machine, but saving and getting a setting doesn't appear to work for me (I tried in Word and Excel). Happy to jump on a call if needed.